### ISSUE TITLE
[Bug]: Volcano Manor Crashing game or corrupting saves

### ISSUE THREAD

**Issue by Stealthy1Dragon** [LukeYui/EldenRingSeamlessCoopRelease#160]
### Discord Username

StealthyDragon#3481

### Describe the issue in detail

I get to volcano manor and it crashes my game but then stuck in the area not able to play the character since I'm stuck spawning there it constantly crashes. Made another character and got to volcano manor for the same thing to happen.

### Steps to recreate the bug

Get to volcano manor, game crashes, restart game stuck unable to leave as you crash immediately (before and after getting a grace). Unable to play may have corrupted the saves.

### Version

1.2.4 (Latest)

### Relevant Crash log

```shell
Faulting application name: eldenring.exe, version: 1.5.0.0, time stamp: 0x6295c83c
Faulting module name: eldenring.exe, version: 1.5.0.0, time stamp: 0x6295c83c
Exception code: 0xc0000005
Fault offset: 0x0000000000234619
Faulting process id: 0x46a0
Faulting application start time: 0x01d87f8c8e44e083
Faulting application path: D:\Games\steamapps\common\ELDEN RING\Game\eldenring.exe
Faulting module path: D:\Games\steamapps\common\ELDEN RING\Game\eldenring.exe
Report Id: 68b3404a-3adb-4781-b71d-98d7e432fad9
Faulting package full name: 
Faulting package-relative application ID:
```

**Comment by Jacobocious** [comment]
I too am having this exact problem, and cannot play that specific character. I am able to load up other characters (not at volcano manor yet). I loaded up there as a guest, my friend was hosting, and only I got disconnected when we fast traveled there. Before I used my effigy, my character was in Liurnia, and then used the effigy and spawned where my friend was at (in Caelid), we then fast traveled to Volcano Manor and he was fine, I crashed. Now when I try to load up my character (before joining anyone's game) I start in Volcano Manor instead of Liurnia, and it just continually crashes ~not sure if this may have something to do with it, where because
